app/lib/import_export_constants.rb

```diff
@@ -283,6 +287,7 @@ module ImportExportConstants # rubocop:todo Metrics/ModuleLength
     closed_at: 'Closure Date',
     monitoring_reason: 'Reason For Closure',
     expected_purge_ts: 'Expected Purge Date',
+    contact_became_case_at: 'Became Case At',
```

Comment:
the label here and the one on line 110 should be consistent, I would probably prefer this one since it's shorter though

Reply:
The other label is what the PO team decided on, so I changed it here: https://github.com/SaraAlert/SaraAlert/pull/1085/commits/5f4d6b583878d607a0f11887bcf0739f09f58946